_posts/2019-05-28-improved-hive-bucketing.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title:  "Improved Hive Bucketing"
+author: David Phillips
+---
+
+The next release of Presto (version 312) adds support for the more
+flexible bucketing introduced in recent versions of Hive. Specifically,
+it allows any number of files per bucket, including zero. This allows
+inserting data into an existing partition without requiring a rewrite
```

Comment:
`This allows inserting data into an existing partition without having to rewrite the entire partition`